platform/src/lib.rs

```diff
@@ -65,11 +73,15 @@ pub trait Platform {
     /// * `out` - Output buffer for issuer name to be written to.
     fn get_issuer_name(&mut self, out: &mut [u8; MAX_CHUNK_SIZE]) -> Result<usize, PlatformError>;
 
-    /// Retrives the issuer's Serial Number
+    /// Retrieves a CMS Content Info's signer identifier
     ///
-    /// The issuer serial number is a big-endian integer which is at-most 20
-    /// bytes. It must adhere to all the requirements of an ASN.1 integer.
-    fn get_issuer_sn(&mut self, out: &mut [u8; MAX_SN_SIZE]) -> Result<usize, PlatformError>;
+    /// This function can simply return an error if the DPE does not support CSRs
+    fn get_signer_identifier<'a>(
```

Comment:
nit: Can you explain in the comment that the platform can choose either SubjectKeyIdentifier or IssuerAndSerialNumber

Reply:
Done